### DISCUSSION TITLE
GitHub metaverse - Bounty DAO

### DISCUSSION THREAD

**Discussion by sergfeldman** [pavlovcik/uad-ui#432] [Research]
> GitHub metaverse - Bounty DAO

Start the Bounty DAO (or Bounties DAO) - the ecosystem of bounties for developers.

There are yield-bearing stablecoins on the market.
We can provide a discount stablecoin (or a discount bond).

Launch the utility token Bounty Dollar - a stablecoin that unlocks discounts. 

Info
https://blog.obyte.org/introducing-discount-stablecoins-6e7b5e9a8fd6
https://en.m.wikipedia.org/wiki/Zero-coupon_bond 
https://ostable.org/create 
TBD

> Bounty DAO ecosystem markets

`Bounties marketplace`

Earn a Bounty Dollar for completing bounties.
"Come code - get paid"

As a developer you
Give 
Your work at a suitable time
Get
Income

As a project you
Give
Tasks for developers-freelancers

Get
The lower total cost of development
The higher overall quality of solutions 

`Discounts marketplace `

Purchase digital products and services with discounts for Bounty Dollar.
"Pay upfront - get discount"

As a developer you
Give
Lock your liquidity
Get 
Access to exclusive discounts for the demanded products 

As a project you
Give
Discounts on your products
Get
New customers from your demographic
More stable revenue 

`Easy integration with traditional payments `

Develop the Bounty DAO in partnership with Visa/MasterCard/Stripe

`The only way to get Bounty Dollar is to deposit USD to the Bounty pool `

The Bounty pool is a Bond-style pool.
After a lockup period, you will get Bounty Dollar with a discount.
The longer the deposit period - the higher discount 
For example, 1 day lock = 0.1% discount, min 1 week, max 1 year, withdraw anytime

For degens providing liquidity to the Bounty pool is similar to Bonds in case they are able to sell Bounty Dollar.

`What to do with black market for the Ubiquity Dollar? `

Direct swap like UCDC-Bounty Dollar prohibited or with high fees?

Research Bounty Dollar as an ERC-3525 token 

Research ERC-3525 use cases
https://foresightnews.pro/article/detail/13775
TBD

**Comment by 0x4007** [comment]
This sounds like what we're already looking to do with the DevPool except you're proposing to make a new token which I'm very hesitant on. We would then have to build liquidity for new markets, again. We should discuss on our research call.

**Comment by sergfeldman** [comment]
Ponzi elements for our products

The purpose of Ponzi elements is to increase activity driven by greed, the anticipation of future revenue, and a risks-on attitude. 

**Ponzi and gamification mechanics:**
- Discounts, cumulative discounts (like in Bonds)
- High APY 
- Gambling, random airdrops, raffles (do-win)
- MLM (do once - get passive income)
- Referrals (invite - get passive income)
- Big or attractive ideas (participate - get a sense of belonging)

**Products and ponzi elements**
Credit system (Dollar, Credits, Bonds) - Discounts, High APY, Big or attractive ideas (utility)
Vault - Gambling, Referrals
Lootbox - Discounts, MLM
